src/modules/booking/components/modal-success/modal-successs.tsx

```diff
@@ -0,0 +1,102 @@
+import { type FC } from 'react';
+import { format } from 'date-fns';
+
+// components
+import { Button } from '@/modules/core/components/button';
+import { Dialog } from '@/modules/core/components/dialog';
+import { Typography } from '@/modules/core/components/typogrpahy';
+import { Icon } from '@/modules/core/components/icon';
+import { InfoBox } from '../modal-success-infobox/info-box';
+
+// style
+import styles from './modal-success.module.scss';
+import { type BookingModalSuccessProps } from './modal-success.interface';
+
+export const BookingModalSuccess: FC<BookingModalSuccessProps> = ({
+  bookingData,
+  ...props
+}) => {
+  const handleModalClose = () => {
+    if (props.onOpenChange) {
+      props.onOpenChange(false);
+    }
+  };
+
+  const formattedDate = format(
+    bookingData?.startTime ? new Date(bookingData?.startTime) : new Date(),
+    `d MMM`
+  );
+
+  const formattedStartTime = format(
+    bookingData?.startTime ? new Date(bookingData?.startTime) : new Date(),
+    'HH:mm'
+  );
+  const formattedEndTime = format(
+    bookingData?.endTime ? new Date(bookingData?.endTime) : new Date(),
+    'HH:mm'
+  );
+
+  const statusSuccess =
+    bookingData?.status === 'PENDING' ? 'Requested' : 'Approved';
+
+  let currency = '';
```

Comment:
Don't use let. If needed you can create parser that will you the correct currency, and use it like this:
`const currency = currencies[bookingData?.serviceProfessional.currency ?? 'default']`
That would be better than using switch. Also you have Currency enum already in prisma, so you can use it as well.